web/src/settings_account.ts

```diff
@@ -239,14 +258,25 @@ export function add_custom_profile_fields_to_settings() {
     custom_profile_fields_ui.initialize_custom_pronouns_type_fields(element_id);
 }
 
-export function hide_confirm_email_banner() {
+export function hide_confirm_email_banner(): void {
     if (!overlays.settings_open()) {
         return;
     }
     $("#account-settings-status").hide();
 }
 
-export function update_privacy_settings_box(property) {
+// TODO: Move these to server_events_dispatch when it's converted to typescript.
+export const privacy_settings_schema = z.enum([
+    "send_stream_typing_notifications",
+    "send_private_typing_notifications",
+    "send_read_receipts",
+    "presence_enabled",
+    "email_address_visibility",
+    "allow_private_data_export",
+]);
+export type PrivacySettings = z.infer<typeof privacy_settings_schema>;
```

Comment:
The type of a single setting should use a singular name, like `PrivacySetting` or `…Key` or `…Name` or `…Property`.

Reply:
I went with `PrivacySettingName`